calibrate.py

```diff
@@ -177,9 +177,8 @@ def recalibrate(files):
     #     if '_recal.Spe' in file:
     #         files.remove(file)
 
-    files_updated = [os.path.splitext(files[i])
-                     [0].replace("_recal", "")
-                     for i in np.arange(len(files))]
+    files_updated = [os.path.splitext(f)[0].replace("_recal", "")
```

Comment:
This variable is only used in the `calibration_table` call below. The `replace` is not changing the length of the file list, it's only changing the name of the files printed in the table. I'm saying the name in the table should be the `_recal` version, not the base version. The `_recal` version is what is analyzed, right?